app/scripts/modules/core/serverGroup/details/multipleServerGroups.controller.js

```diff
@@ -29,14 +31,21 @@ module.exports = angular.module('spinnaker.core.serverGroup.details.multipleServ
         return descriptor;
       };
 
-      let confirm = (submitMethod, verbs) => {
+      let confirm = (submitMethodName, verbs) => {
         let descriptor = getDescriptor(),
             monitorInterval = this.serverGroups.length * 1000;
         let taskMonitors = this.serverGroups.map(serverGroup => {
+          let provider = serverGroup.provider || serverGroup.type;
+          let submitter = serviceDelegate.hasDelegate(provider, 'serverGroup.submitter') ?
```

Comment:
Good idea, the fallback/default behavior is neater in that case. I'll make some changes.